build_artifacts.py

```diff
@@ -443,8 +443,13 @@ def _convert_polygon(polygon):
             for coordinate in polygon:
                 pairs = []
                 for pair in coordinate:
-                    lon_lat = list(proj(pair[0], pair[1], inverse=True))
-                    pair = lon_lat + pair[2:] if len(pair) >= 2 else lon_lat
+                    lon, lat = pair[0], pair[1]
```

Comment:
I think you can also do `lon, lat = pair[0:2]` if you want

Reply:
Oh smart, will fix.